recipes/libgpiod/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "libgpiod" %}
+{% set version = "2.1.2" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://git.kernel.org/pub/scm/libs/{{ name }}/{{ name }}.git/snapshot/{{ name }}-{{ version }}.tar.gz
+  sha256: b1bdf1e3f75238695f93e442062bafc069170f2bf4f0cd4b8e049ca67131a1f0
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+outputs:
+  - name: {{ name }}
+    script: build_base.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - make
+        - autoconf
+        - autoconf-archive
+        - pkg-config
+        - automake
+        - libtool
+  - name: gpiod-python
+    script: build_py.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - autoconf
+        - swig
+      host:
+        - {{ pin_subpackage('libgpiod', exact=True) }}
+        - wheel
+        - setuptools
+        - python >=3.9
+        - pip
+        - packaging
+      run:
+        - python >=3.9
+        - {{ pin_subpackage('libgpiod', exact=True) }}
+
+about:
+  home: https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/about
+  summary: C library and tools for interacting with the linux GPIO character device (gpiod stands for GPIO device)
+  license: LGPL-2.1-or-later AND GPL-2.0-or-later
+  license_file:
+    - LICENSES/LGPL-2.1-or-later.txt
+    # used in gpio-tools
```

Comment:
This is only relevant for `libgpiod`. Is there a way to set `gpiod-python` to only `LGPL-2.1-or-later`?

Reply:
Yes, copy the about section to the outputs.

Reply:
and which license goes in the about section of the recipe? https://github.com/conda-forge/staged-recipes/pull/26443#issuecomment-2128715999

Reply:
The full one `license: LGPL-2.1-or-later AND GPL-2.0-or-later`